eng/tools/lint-diff/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@azure-tools/lint-diff",
+  "private": true,
+  "type": "module",
+  "main": "dist/src/lint-diff.js",
+  "bin": {
+    "lint-diff": "cmd/lint-diff.js"
+  },
+  "scripts": {
+    "build": "tsc --build",
+    "test": "vitest",
+    "test:ci": "vitest run --coverage --reporter=verbose",
+    "clean": "rm -rf dist && rm -rf node_modules"
+  },
+  "engines": {
+    "node": ">= 18.0.0"
+  },
+  "dependencies": {
```

Comment:
Ensure all deps and devDeps are aligned with similar tools in the repo

Reply:
Added to https://github.com/Azure/azure-rest-api-specs/issues/33302